src/braket/jobs/local/local_job_container.py

```diff
@@ -230,13 +240,16 @@ def copy_from(self, source: str, destination: str) -> None:
 
     def run_local_job(
         self,
-        environment_variables: Dict[str, str],
+        environment_variables: dict[str, str],
     ) -> None:
         """Runs a Braket Hybrid job in a local container.
 
         Args:
-            environment_variables (Dict[str, str]): The environment variables to make available
+            environment_variables (dict[str, str]): The environment variables to make available
                 as part of running the hybrid job.
```

Comment:
is this formatting we are following?

Reply:
Yes, subsequent lines should be indented

Reply:
Following the convention from https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html